spec/support/standard_runner.rb

```diff
@@ -21,10 +24,22 @@ def issues(output = @engine_output)
   end
 
   def run_engine(config = nil)
+    @config = config
```

Comment:
set as an instance variable now so _if_ config arg was passed into this (which it is in at least one spec in this repo) then we use that when generating the `registry` value instead of a blank config object (see below)